examples/remove-local-config-resources-simple/README.md

```diff
@@ -0,0 +1,52 @@
+# # remove-local-config-resources: Simple Example
```

Comment:
Only one `#` please

Reply:
I'd suggest to name it `remove-local-config-resources: Imperative Example`.

Reply:
Changed. And renamed the folder as well.